app/src/androidTest/java/ch/epfl/sdp/blindly/main_screen/ViewPagerAdapterTest.kt

```diff
@@ -6,7 +6,7 @@ import org.hamcrest.Matchers.equalTo
 import org.junit.Test
 import org.mockito.Mockito.mock
 
-private const val COUNT = 4
+private const val COUNT = 5

```

Comment:
I don't know if it will create a conflict, but isn't there supposed to be only 4 fragments?
The match, the my matches, the profile and the activity suggestion tabs? (Since we agreed to move the map to the
my match)

Reply:
I will deal with this before #209